protocol/kafka_segmentio/v2/sender_test.go

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package kafka_segmentio
+
+import (
+	"testing"
+)
+
+func TestSenderWithKey(t *testing.T) {
+	// topic := "aaa"
+	// writer := kafka.NewWriter(kafka.WriterConfig{
```

Comment:
any way to make this work in memory? or perhaps it should be an e2e test and we can launch the broker like we do in the other tests?